internal/report/snapshot_reporter.go

```diff
@@ -198,7 +201,7 @@ func (s *SnapshotReporter) SaveMetrics(componentID string, md pmetric.Metrics) {
 }
 
 // prepRequestPayload based on the pipelineType will return a marshaled proto of the OTLP data types for the componentID
-func (s *SnapshotReporter) prepRequestPayload(componentID, pipelineType string) (payload []byte, err error) {
+func (s *SnapshotReporter) prepRequestPayload(componentID, pipelineType string, searchQuery *string, minimumTimestamp *time.Time) (payload []byte, err error) {
```

Comment:
nit: Same here the `searchQuery` and `minimumTimestamp` variable names are not consistent.